src/Widgets/DataGroupWidget/DataGroupWidgetClass.ts

```diff
@@ -0,0 +1,8 @@
+import { provideWidgetClass } from 'scrivito'
+
+export const DataGroupWidget = provideWidgetClass('DataGroupWidget', {
+  attributes: {
+    content: 'widgetlist',
```

Comment:
Kris and I thought about this too. But we decided to keep them separate for now, since a "DataGroupWidget" will (with 1.40) also get a different widget "container" color, where as the " GroupWidget" will stay a "plain" container widget.

Reply:
Actually my expectation is that only widgets with data configured get another color:
* No datalocator set: yellow
* Datalocator set: purple

Reply:
At least with the current `master` this is not the case:

![Group without data](https://github.com/Scrivito/scrivito-portal-app/assets/86275/353fec94-f1be-43e5-b0ae-e654ef895b37)

It always renders a purple border, regardless of the datalocator being set or not.